bots/pr/src/main/java/org/openjdk/skara/bots/pr/PullRequestCommandWorkItem.java

```diff
@@ -228,4 +228,18 @@ public String toString() {
     public String workItemName() {
         return "pr-command";
     }
+
+    /**
+     * This method returns all the comments in the pr including comments in reviews(review body)
+     */
+    private List<Comment> getAllComments() {
+        // Regular comments
+        List<Comment> comments = new ArrayList<>(prComments());
```

Comment:
It's not really important, but you could use Streams.concat() instead of creating two intermediate lists.

Reply:
It would be better